### PR TITLE
This PR add 'master' key to do request for all client

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR add 'master' key to do request for all client

## How to test

Here are the following steps to test this pull request:

- Add `master` key in `chaos/clients_codes.json` 
- If you use token of master you should not have `401 response`

### CODE REVIEW

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

File: README.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ If you want to add more security, you can add a file chaos/clients_tokens.json w
 client_code should be the same as the value of X-Customer-Id header in HTTP request and token should be the same as the value of Authorization header in HTTP request
 If the file doesn't exist, the security will be disabled.
 
+When you add `master` key in file, tokens will be accepted for all request for any `client_code`. Be careful, `client_code` should be exist in file.
```
[the remark this answers is not in the document]
Comment (pthegner) [reply README.md]: You can add a 'master' key in the file. It will allow you to access all ressources for all clients.

File: README.md [hunk 2]
```diff
@@ -129,6 +129,8 @@ If you want to add more security, you can add a file chaos/clients_tokens.json w
 client_code should be the same as the value of X-Customer-Id header in HTTP request and token should be the same as the value of Authorization header in HTTP request
 If the file doesn't exist, the security will be disabled.
 
+You can add a 'master' key in the file. It will allow you to access all ressources for all clients.
```
Comment (dvdn) [inline README.md]: resources

**Pull request merged**